### ISSUE TITLE
battery is not being shown on Surface Book 1

### ISSUE THREAD

**Issue by IleanaAldama** [martinnyborg/arch-surface-jakeday-kernel#1]
Hello, thanks for providing this PKGBUILD,
I installed  the 3 generated packages
- linux-surface-jakeday 4.17.3-1
- linux-surface-jakeday-docs 4.17.3-1
- linux-surface-jakeday-headers 4.17.3-1  

After running makepkg and pacman -U
Everything seems working fine, except for the battery, is not show on /proc/api/battery or in powertop

**Comment by martinnyborg** [comment]
Hey @ChrisAldama 

The battery not showing is a known issue with the jakeday kernel https://github.com/jakeday/linux-surface/issues/195

I'll update this package once a fix has been pushed to https://github.com/jakeday/linux-surface

**Comment by IleanaAldama** [comment]
Thanks but I think that issue happens only with the Surface Book 2, in my case I own a Surface Book 1 and the battery shows on kernel 4.16.9 but not in 4.17.3, 

`[ileana@surface ~]$ upower -i /org/freedesktop/UPower/devices/battery_BAT0
  native-path:          (null)
  power supply:         no
  updated:              mié 31 dic 1969 16:00:00 PST (1530913065 seconds ago)
  has history:          no
  has statistics:       no
  unknown
    warning-level:       unknown
    battery-level:       unknown
    icon-name:          '(null)'

[ileana@surface ~]$ upower -i /org/freedesktop/UPower/devices/battery_BAT1
  native-path:          BAT1
  vendor:               SMP
  model:                X906972
  serial:               4834
  power supply:         yes
  updated:              vie 06 jul 2018 16:36:21 CDT (94 seconds ago)
  has history:          yes
  has statistics:       yes
  battery
    present:             yes
    rechargeable:        yes
    state:               discharging
    warning-level:       none
    energy:              13.282 Wh
    energy-empty:        0 Wh
    energy-full:         17.452 Wh
    energy-full-design:  17.902 Wh
    energy-rate:         1.822 W
    voltage:             8.074 V
    time to empty:       7.3 hours
    percentage:          76%
    capacity:            97.4863%
    technology:          lithium-ion
    icon-name:          'battery-full-symbolic'
  History (charge):
    1530912981  76.000  discharging
  History (rate):
    1530912981  1.822   discharging

[ileana@surface ~]$ uname -a
Linux surface 4.16.9-1-surface4 #5 SMP PREEMPT Thu May 17 08:51:32 CST 2018 x86_64 GNU/Linux
[ileana@surface ~]$ 
`

**Comment by martinnyborg** [comment]
I see what you mean. 
Did you compile your previous kernel? And if so, can share the configuration file? I'm guessing I skipped a module.

**Comment by ZHG2017** [comment]
Hi, if I understand the issue correctly, the battery status  can be correctly displayed on surface book but not on a surface book 2, isn't it ? I wonder if I should buy a surface book instead of surface book 2 as I really need to use linux for some stuff

**Comment by martinnyborg** [comment]
@ZHG2017 That's correct. It is a known issue of the jakeday kernel that is being worked on. Hopefully it will be fixed in the near future.

**Comment by ZHG2017** [comment]
@martinnyborg OK, I see, thanks a lot for the confirmation and let's hope this will be fixed soon
